client/service_logs.go

```diff
@@ -7,13 +7,12 @@ import (
 	"net/url"
 	"time"
 
-	"github.com/moby/moby/api/types/container"
 	"github.com/moby/moby/client/internal/timestamp"
 )
 
 // ServiceLogs returns the logs generated by a service in an [io.ReadCloser].
 // It's up to the caller to close the stream.
-func (cli *Client) ServiceLogs(ctx context.Context, serviceID string, options container.LogsOptions) (io.ReadCloser, error) {
+func (cli *Client) ServiceLogs(ctx context.Context, serviceID string, options ContainerLogsOptions) (io.ReadCloser, error) {
```

Comment:
Wondering if we should create a separate `ServiceLogOptions` struct to allow them to diverge, but also if we want to do the functional options in stages.

Reply:
+1, separate service log options. Can be follow-up PR though if you want.